### ISSUE TITLE
Create a subdomain for each mixed content error

### ISSUE THREAD

**Issue by lgarron** [lgarron/badssl.com#154]
https://github.com/lgarron/badssl.com/issues/152#issue-137988831

We already have a lot of these:

- `mixed-css.badssl.com`
- `mixed-font.badssl.com`
- `mixed-form.badssl.com`
- `mixed-frame.badssl.com`
- `mixed-image.badssl.com`
- `mixed-password.badssl.com`
- `mixed-script.badssl.com`

**Comment by marumari** [comment]
My opinion is that we kill all of them in favor of them just being subdirectories.

**Comment by lgarron** [comment]
I disagree, because it helps to have the origin reflect the problem if you want to test a specific one of these.

**Comment by marumari** [comment]
I don't get how:

https://mixed-script.badssl.com/ is really that much more reflective that https://badssl.com/mixed/script/

How about we have them 301 redirect instead? Then it shouldn't affect any of the existing tests afaik.

**Comment by lgarron** [comment]
Some UI only shows the current origin/domain.

1) It's very convenient for these UI strings to reinforce what you think you're testing.
2) People seeing these URLs will find them easy to remember and type into different devices and browsers.

<img width="310" alt="screen shot 2016-03-02 at 12 59 38" src="https://cloud.githubusercontent.com/assets/248078/13475259/b21c2820-e076-11e5-837b-d3ab666c1bdf.png">

**Comment by lgarron** [comment]
I don't know about you, but I often work with random community members and project testers, and I'd like the origin to be as clear as possible.

A lot of the `TEST=` documentation in our CLs also refers specifically to `mixed.badssl.com` and `mixed-script.badssl.com`, and I'd like to avoid breaking those unnecessarily.

**Comment by marumari** [comment]
I'm a disagreeable old curmudgeon, and so random community tends to avoid me as much as possible.  :P

I honestly don't feel that strongly about this, so we can leave them around if you find it useful.

**Comment by lgarron** [comment]
Alright, we can figure it out later. I'll hold off on adding any new ones for now.

**Comment by Hoverbear** [comment]
@marumari Is your complaint more about cluttering the homepage or the configurations?

**Comment by marumari** [comment]
The configurations, mostly.  It's a single link either way, off the home page.  :)
